src/components/Modal/README.md

```diff
@@ -92,11 +92,24 @@ See [API](#api) for all available options.
 - **Modal actions** should correspond to the modal purpose, too. E.g. “Delete”
   tells better what happens rather than “OK”.
 
-- Modal **automatically focuses the first non-disabled form field** by default
-  which allows users to confirm the modal by hitting the enter key. When no
-  field is found then the primary button (in the footer) is focused. To turn
+- While native `<dialog>` (that is used under the hood) can be present in DOM,
+  modal is a more feature-rich component that provides more control over the
```

Comment:
I'd consider using capital `M` whan speaking about RUI component:

```suggestion
  Modal is a more feature-rich component that provides more control over the
```

This way we can distinguish our component from a generic design pattern.

Reply:
I checked the rest of the documentation, I must say that this is the style we use among the documentation. So, if you want to have it as you purpose, I would solve it in separate issue as this is something not related to this change.

Personally, I have not opinion on this. If we talk about the inner component, we typically  emphasize it by using HTML tags rather than using Modal vs modal, Button vs button. I would not say intentionally, but I can see the parent there.

What do you think @mbohal and @adamkudrna ?

Reply:
No problem, feel free to dismiss it.